test/org/javarosa/xform/parse/XFormParserTest.java

```diff
@@ -331,6 +333,25 @@ public void parseFormWithTwoModels() throws IOException {
         assertEquals("first-model", firstModelInstanceId);
     }
 
+    @Test
+    public void parseFormWithSetValueAction() throws IOException {
+        // Given & When
+        ParseResult parseResult = parse(r("form-with-setvalue-action.xml"));
+        FormDef formDef = parseResult.formDef;
+
+        // dispatch 'xforms-ready' action (Action.EVENT_XFORMS_READY)
+        formDef.initialize(true, new InstanceInitializationFactory());
+
+        // Then
+        assertEquals(formDef.getTitle(), "SetValue action");
+        assertEquals("Number of error messages", 0, parseResult.errorMessages.size());
```

Comment:
Just a suggestion: Extract method -> assertNoParseErrors
Or maybe assertNumParseErrors(int).

Reply:
I like this suggestion. I went with `assertNoParseErrors` as the IDE was whining that the expected number of errors is always 0 -_-